### PR TITLE
Places a monument to our failure to defeat tg.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
add: Added the giant stone ass as a monument to our failure in arrivals.
imageadd: Statue sprite by Satoun
/:cl:

While making the game unplayable for a few players in a time honoured coder shitpost it was rightly denied. However us losing to tg cannot go unforgotten lest we fail again next time. 

Like the clown statue it makes a noise when clicked or pushed, in this case its a fart but it plays that massive fart with reverb noise when deconstructed.

I can delete mention of eagle, praisenarise and pyko directly if you feel its a bit mean.

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
If I had the time I would make a set of ass bagpipes.

image

File: hippiestation/code/game/objects/structures/statues.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/obj/structure/statue/sandstone/asstue
+	name = "Giant stone ass."
+	desc = "On Saturday, August 4th, 2018 the toolbox was thrown down by the people of tgstation,\
+	 three randomly selected hippies took up the challengeand stood poised to fight the tg menace. \
```
Comment (ghost) [inline hippiestation/code/game/objects/structures/statues.dm]: challengeand

**Comment by praisenarsie** [comment]
i'm perfectly fine with the mention of me

File: hippiestation/code/game/objects/structures/statues.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/obj/structure/statue/sandstone/asstue
+	name = "Giant stone ass."
+	desc = "On Saturday, August 4th, 2018 the toolbox was thrown down by the people of tgstation,\
+	 three randomly selected hippies took up the challenge and stood poised to fight the tg menace. \
+	 They failed miserably and bought shame upon our once proud station. We have made this giant ass \
+	 statue to commemorate their failure and the giant ass they made of us. \
+	 Pyko, Praisenarsie, EagleEyes: their names etched upon this stone butt for all eternity."
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: i prefer to be mentioned as Frozenguy5

File: hippiestation/code/game/objects/structures/statues.dm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	icon = 'hippiestation/icons/obj/statue.dmi'
+	icon_state = "asstue"
+
+/obj/structure/statue/sandstone/asstue/proc/toot()
+	playsound(src.loc, 'hippiestation/sound/effects/fart.ogg', 100, 1)
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: remove .loc

File: hippiestation/code/game/objects/structures/statues.dm [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+/obj/structure/statue/sandstone/asstue/Bumped(atom/movable/AM)
+	toot()
+	..()
+
+/obj/structure/statue/sandstone/asstue/attackby(obj/item/W, mob/user, params)
+	toot()
+	return..()
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: isn't a space required here or is byond not that strict

Comment (GuyonBroadway) [reply hippiestation/code/game/objects/structures/statues.dm]: I mean it compiled ok and it worked in game.

Comment (ghost) [reply hippiestation/code/game/objects/structures/statues.dm]: even if it isn't it should be standarisied to `return ..()` like everywhere else in the code

**Comment by Eagle-Eyess** [comment]
Can I see a pic of said sprite

**Comment by CitrusGender** [comment]
image
not bad

**Comment by ghost** [comment]
add a storage component to it so people can hide stuff inside the butt

**Comment by Carbonhell** [comment]
I'm +1ing this ONLY because of the butt

**Comment by DaydreamPuppeh** [comment]
Is that butt based on Billy's butt?

**Comment by ghost** [comment]
make a monument for the times i accidentally fucked up the code

**Comment by ghost** [comment]
+1 butt

**Comment by GuyonBroadway** [comment]
Wow this doesn't fucking conflict since them chemistry changes somehow.

File: hippiestation/code/game/objects/structures/statues.dm [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+/obj/structure/statue/sandstone/asstue/attack_hand(mob/user)
+	toot()
+	.=..()
+
+/obj/structure/statue/sandstone/asstue/attack_paw(mob/user)
+	toot()
+	..()
+
+/obj/structure/statue/sandstone/asstue/deconstruct(disassembled = TRUE)
+	playsound(src.loc, 'hippiestation/sound/effects/superfart.ogg', 200, 1)
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: src

File: hippiestation/code/game/objects/structures/statues.dm [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	toot()
+	..()
+
+/obj/structure/statue/sandstone/asstue/attackby(obj/item/W, mob/user, params)
+	toot()
+	return ..()
+
+/obj/structure/statue/sandstone/asstue/attack_hand(mob/user)
+	toot()
+	.=..()
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: . = ..()

File: hippiestation/code/game/objects/structures/statues.dm [hunk 7]
```diff
@@ -0,0 +1,32 @@
+/obj/structure/statue/sandstone/asstue
+	name = "Giant stone ass."
```
Comment (praisenarsie) [inline hippiestation/code/game/objects/structures/statues.dm]: remove capital and period

**Comment by yoyobatty** [comment]
What even..

File: hippiestation/code/game/objects/structures/statues.dm (hunk 6 above)
Comment (GuyonBroadway) [reply hippiestation/code/game/objects/structures/statues.dm]: It says .=..()

Thats exactly what it already says.

Comment (ghost) [reply hippiestation/code/game/objects/structures/statues.dm]: spaces

Comment (praisenarsie) [reply hippiestation/code/game/objects/structures/statues.dm]: it's the standard!!!!!!

**Comment by ghost** [comment]
-1

This never happened.

**Comment by GuyonBroadway** [comment]
We must bear our shame lest it happen again.

**Comment by praisenarsie** [comment]
should this be anchored?

**Comment by GuyonBroadway** [comment]
It is anchored mapside.

**Comment by JohnGinnane** [comment]
Is this ready?

**Comment by GuyonBroadway** [comment]
Yes

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**